### PR TITLE
Update README to include features, link to manual

### PR DESCRIPTION
**Pull request by dLobatog**
**Problem**
People come to the repo and don't know how to install the project,
what does it exactly do, etc.. People ask in #theforeman IRC channel
about basic help on how to install the plugin because they didn't
see any documentation in the README.

**Solution**
Add a few basic information screenshots, points of contact, and links
to the manual to ensure new users don't get lost.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,36 +5,28 @@
...
-
-3. ``bundle exec jekyll serve`` to see the rendered changes locally
-
-4. ``bundle exec rake publish`` to publish the changes to Github pages
+Check the Foreman manual [remote execution section](http://theforeman.org/plugins/foreman_remote_execution/)
 
 ## Links
 
-* [the project page](http://theforeman.github.io/foreman_remote_execution/)
-* [the issue tracker](http://projects.theforeman.org/projects/foreman_remote_execution)
```
Comment (ares) [inline README.md]: could we keep the tracker link please?

**Comment by ares** [comment]
one nit, otherwise good to merge I think

File: README.md (hunk 1 above)
Comment (iNecas) [inline README.md]: +1

**Comment by dLobatog** [comment]
Sorry, I removed the traker link by mistake. Updated

**Comment by iNecas** [comment]
ACK - I'm ok with not enforcing an open issue for README changes :)

**Comment by dLobatog** [comment]
@iNecas It's not enforced in core even for changes that don't go in releases, license/readme, transifex pulling strings :smile:

**Comment by iNecas** [comment]
Thanks @dLobatog !

**Pull request merged**